backend/controllers/retentionReasons.js

```diff
@@ -11,25 +10,23 @@ const retentionReasonsFactory = (elite2Api, dataComplianceApi, logError) => {
 
   const getOffenderUrl = offenderNo => `${dpsUrl}offenders/${offenderNo}`
 
-  const formatDate = date => date && formatTimestampToDate(date)
-
-  const formatDateTime = dateTime => dateTime && formatTimestampToDateTime(dateTime)
-
   const getFormAction = offenderNo => `/offenders/${offenderNo}/retention-reasons`
 
   const getLastUpdate = existingRecord =>
-    existingRecord && {
-      version: existingRecord.etag,
-      timestamp: formatDateTime(existingRecord.modifiedDateTime),
-      user: existingRecord.userId,
-    }
+    existingRecord
+      ? {
+          version: existingRecord.etag,
+          timestamp: formatTimestampToDateTime(existingRecord.modifiedDateTime),
+          user: existingRecord.userId,
+        }
+      : {}
 
   const flagReasonsAlreadySelected = (retentionReasons, existingRecord) => {
-    const existingReasons = existingRecord && existingRecord.retentionReasons
+    const existingReasons = existingRecord.retentionReasons || []
```

Comment:
this isn't entirely the same as previously `existingRecord` could have been undefined.. 

could provide a default param: `(retentionReasons, existingRecord = {}) => {`
or maybe pass through only the existing reasons and default them..